Stubbornium/StubbornWebElement.cs

```diff
@@ -182,6 +182,8 @@ public void Assert(Func<Func<IWebElement>, bool> assertion, string logMessage)
             int attemptNo = 0;
             while (true)
             {
+                configuration.MiddleDoActions.ForEach(action => action(browser));
```

Comment:
I wonder if it wouldn't be better to move the line 179 (` configuration.BeforeDoActions.ForEach(action => action(browser));) here, and move this line to line 188, so that it's executed between `seleniumAction` and `wait.Until(expectedCondition)`. 

This way we could define a conditions that must be met right before and right after each intearction/